app/build.gradle

```diff
@@ -113,6 +113,7 @@ ext {
     mockito = "2.18.3"
     mockitoKotlin = "2.0.0"
     commonsMath = "3.6.1"
+    commonsCodec = "1.13"
```

Comment:
is this just used for `sha1` and `sha256`? if so, we could do that without pulling in further dependencies. `okio` is already a transitive dependency of `okhttp`, so we could use `ByteString`

    val byteString = ByteString.encodeString(helloWorldText, Charset.defaultCharset())
    val sha1 = byteString.sha1().hex()
    val sha256 = byteString.sha256().hex()

Reply:
I was a fan of this idea however now that I've played with okio I'm getting deprecation warnings everywhere, their docs are out-of-date and copying a ByteArray to a ByteString to get a sha256 feels icky (we use it on files which have been up to 10mb in the past). I prefer commons codec but if you feel strongly about excluding it, I can write the sha1 method (we already had a sha256 previously). Let's discuss tomorrow.